src/components/Common/ExternalAnchor.js

```diff
@@ -13,8 +12,4 @@ const ExternalAnchor = ({ children, openInNewWindow = true, ...props }) => {
   )
 }
 
-ExternalAnchor.propTypes = {
-  title: PropTypes.string.isRequired
```

Comment:
There are a few more areas in the codebase you've missed that I added. Search the repo for 
`title: PropTypes.string.isRequired` and you'll find em

Reply:
I think I got them all. I left `CaseStudyPreview.js` propTypes as there are loads of them. We may want to check this validation.